engine/gamesys/src/gamesys/components/comp_collection_proxy.cpp

```diff
@@ -432,13 +481,13 @@ namespace dmGameSystem
 
         if (proxy->m_Collection != 0)
         {
-            LogMessageError(message, "The collection %s could not be loaded since it was already.", proxy->m_Resource->m_DDF->m_Collection);
+            LogMessageError(message, "The collection %s could not be loaded since it was already.", GetCollectionResorcePath(proxy));
```

Comment:
Since it was already what? loaded? weird language
(I know it's not your change, but might as well fix it since you are in here)